src/components/grouped-character/components.test-pw.tsx

```diff
@@ -0,0 +1,533 @@
+import React, { useState } from "react";
```

Comment:
**question/suggestion** just a general question on this file, the exported components and their use within `grouped-character.pw.tsx`, could we, instead just have one or two general components like `GroupedCharacterComponent`, with set separators, groups, values etc. make sure its controlled and then just pass props to them in `grouped-character.pw.tsx` for example just pass the `labelInline` prop for the labelInline accessibility test etc. This would cut down on the number of imports within the file, and reduce the amount of test components needed here too. Just a thought 👍